property/index.js

```diff
@@ -166,3 +161,62 @@ module.exports = yeoman.generators.Base.extend({
 
   saveProject: actions.saveProject
 });
+
+function coerceDefault(def, value) {
```

Comment:
Is `def` a definition? If so, I would change it to that. Not a big deal but the more we "spell things out" in the code, the easier it is to ask a community member to make a fix for us!